conformance_tests/sysman/test_sysman_device/src/test_sysman_device.cpp

```diff
@@ -110,13 +110,17 @@ static void run_child_process(const std::string &device_hierarchy) {
 
 TEST_F(
     SYSMAN_DEVICE_TEST,
-    GivenDeviceHierarchyModeCombinedWhenSysmanDeviceUUIDsAreRetrievedThenSysmanAndCoreDeviceUUIDsAreMatched) {
+    GivenDeviceHierarchyModeCombinedAndSysmanEnableMacroIsDisabledWhenSysmanDeviceUUIDsAreRetrievedThenSysmanAndCoreDeviceUUIDsAreMatched) {
```

Comment:
GivenHeirarchyModeCombindedAndSysmanEnableMacroDisabledThenUUIDFromCoreAndSysmanMatches

Reply:
Done. Used Env instead of Macro. Does that look fine ?